runtime/compiler/env/j9method.h

```diff
@@ -596,6 +597,7 @@ class TR_ResolvedRelocatableJ9Method : public TR_ResolvedJ9Method
 
    virtual bool                  storeValidationRecordIfNecessary(TR::Compilation * comp, J9ConstantPool *constantPool, int32_t cpIndex, TR_ExternalRelocationTargetKind reloKind, J9Method *ramMethod, J9Class *definingClass=0);
 
+   static void                          setAttributeResult(bool, bool, uintptr_t, int32_t, int32_t, int32_t, TR::DataType *, bool *, bool *, bool *, void ** );
```

Comment:
Can you fix the spacing so that `setAttributeResult` lines up with the adjacent methods?